playground/web_bindings/Cargo.toml

```diff
@@ -9,12 +9,15 @@ crate-type = ["cdylib", "rlib"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-parser = {version= "*", path = "../../parser/"}
+parser = { version = "*", path = "../../parser/" }
 thiserror = "1.0.38"
-core = {version= "*", path = "../../core/", default-features = false, features = ["web"]}
+core = { version = "*", path = "../../core/", default-features = false, features = ["web"] }
 wasm-bindgen = "0.2.63"
 # The `console_error_panic_hook` crate provides better debugging of panics by
 # logging them with `console.error`. This is great for development, but requires
 # all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
 # code size when deploying.
-console_error_panic_hook = { version = "0.1.6" }
\ No newline at end of file
+console_error_panic_hook = { version = "0.1.6" }
+
+[features]
+default = ["core/bundle_std_packages"]
```

Comment:
Same deal here, the bindings would not need their own features, they can just choose directly when they import core.